lib/zstd.h

```diff
@@ -1053,6 +1053,7 @@ ZSTDLIB_API size_t ZSTD_DCtx_refPrefix(ZSTD_DCtx* dctx,
  *  Note that object memory usage can evolve (increase or decrease) over time. */
 ZSTDLIB_API size_t ZSTD_sizeof_CCtx(const ZSTD_CCtx* cctx);
 ZSTDLIB_API size_t ZSTD_sizeof_DCtx(const ZSTD_DCtx* dctx);
+ZSTDLIB_API size_t ZSTD_sizeof_DCtxWindow(const ZSTD_DCtx* dctx);
```

Comment:
No new library symbol is allowed at "stable" status directly at creation.
When a new symbol is needed, it starts its life as "experimental" (below section)